src/main/java/domain/Game.java

```diff
@@ -0,0 +1,33 @@
+package domain;
+
+import domain.card.Card;
+import domain.card.Deck;
+import domain.user.Dealer;
+import domain.user.Participant;
+import domain.user.Players;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Game {
+    public static final int FIRST_DRAW_NUMBER = 2;
+
+    private Deck deck;
+
+    public Game(Deck deck) {
+        this.deck = deck;
+    }
+
+    public void drawFirst(Players players, Dealer dealer) {
+        List<Card> drawCards = new ArrayList<>();
+        for (int index = 0; index < players.size() * FIRST_DRAW_NUMBER; index++) {
+            drawCards.add(deck.deal());
+        }
+        players.firstDraw(drawCards);
+        dealer.drawFirst(deck.deal(), deck.deal());
+    }
```

Comment:
```suggestion
    public void drawFirst(Players players) {
        List<Card> drawCards = new ArrayList<>();
        for (int index = 0; index < players.size() * FIRST_DRAW_NUMBER; index++) {
            drawCards.add(deck.deal());
        }
        players.firstDraw(drawCards);
    }
```
Players에 딜러도 포함된다면 깔끔한 코드를 가져갈 수 있을 것 같아요.